web/src/components/navbar/mobile/NavBarMobileQuickLinksSlideOutMenu.test.tsx

```diff
@@ -0,0 +1,173 @@
+import { NavBarMobileQuickLinksSlideOutMenu } from "@/components/navbar/mobile/NavBarMobileQuickLinksSlideOutMenu";
+import { getMergedConfig } from "@/contexts/configContext";
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+
+describe("<NavBarMobileQuickLinksSlideOutMenu />", () => {
+  const Config = getMergedConfig();
+
+  window.open = jest.fn();
```

Comment:
posting same comment as desktop
pebble - we've turned open into a jest function. Any reason we are not resetting mocks after each test. Any reason why we are not reverting the open fn back to what it previously was?